lib/licensee/license.rb

```diff
@@ -82,7 +104,14 @@ def initialize(key)
 
     # Path to vendored license file on disk
     def path
-      @path ||= File.expand_path "#{@key}.txt", Licensee::License.license_dir
+      foundpath=nil
+      Licensee::License.license_dirs().each do |dir|
+        foundpath = File.expand_path "#{@key}.txt", dir
+        if(File.exist?(foundpath))
+          break
+        end
+      end
+      @path = foundpath
     end
```

Comment:
Not a huge fan of having to search for file existence across multiple directories. Is there an alternate way we could do this?